WordPress/src/main/java/org/wordpress/android/ui/posts/PostsListFragment.java

```diff
@@ -90,6 +93,7 @@
     private boolean mIsFetchingPosts;
     private boolean mShouldCancelPendingDraftNotification = false;
     private int mPostIdForPostToBeDeleted = 0;
+    private boolean mDoReplayPosition = false;
```

Comment:
Relying on `mDoReplayPosition` seems a little messy. I *think* we can drop the need for it if we change the position manager to only restore the position if non-zero, then reset the position to zero once restored. Ex:

```
public void restoreScrollOffset(RecyclerView recyclerView) {
    if (mRVPosition > 0 || mRVOffset > 0) {
        ((LinearLayoutManager) recyclerView.getLayoutManager())
                .scrollToPositionWithOffset(mRVPosition, mRVOffset);
    }
    mRVPosition = 0;
    mRVOffset = 0;
}
```